.angular-cli.json

```diff
@@ -9,7 +9,7 @@
       "outDir": "dist",
       "assets": [
         "assets",
-        "favicon.ico"
+        "favicon.io"
```

Comment:
I think this re-name is a mistake. There is a favicon-generating service whose URL is favicon.io, but the files they generate still have the .ico extension.

You can still swap out the favicon (I make no pretense to knowing much about art direction) but leave the file extension ".ico" everywhere.